test/integration/test_cases/features/data_input/object_records.mjs

```diff
@@ -0,0 +1,31 @@
+const data = {
+    series: [
+        {
+           name: "Dim0",
+           type: "dimension",
+        },
+        {
+            name: "Dim1",
+            type: "dimension",
+        },
+        {
+            name: "Meas",
+            type: "measure",
+        }
+    ],
+    records: [
+        { Dim0: "a", Dim1: "A" },
```

Comment:
@simzer Please add a test case with non-existing series names too. (If I understand correctly, it won't cause any problems because in the `recordObjectToArray` function, the `forEach` loop goes through the previously existing `seriesList`, so any new series will be ignored.)

Reply:
... and another without existing dimension, for example without Dim1.

Reply:
done